tools/SetupFlow/DevHome.SetupFlow/ViewModels/DevDriveViewModel.cs

```diff
@@ -194,6 +195,7 @@ public double MaximumAllowedSize
     {
         get
         {
+            ResetErrors();

```

Comment:
```suggestion
            ValidateDriveSize();
```
I took a look at these again. Actually instead of both of these using ResetErrors, we should use ValidateDriveSize() for both this one and the below one. This will remove the size validation error without affecting any other validation.